kolibri/plugins/pdf_viewer/assets/src/views/SideBar/index.vue

```diff
@@ -146,7 +146,7 @@
 
   .pdf-sidebar {
     overflow-y: auto;
-    box-shadow: inset -1px 2px 8px rgba(0, 0, 0, 0.16);
+    @extend %dropshadow-2dp;
```

Comment:
And here, again same as [SearchFiltersPanel/index.vue](https://github.com/learningequality/kolibri/pull/12630/files#diff-16a6ba2aaca47edd01098e6c42e746e77f4c247dc44dd2f9ff2559b128764c21)